Tools/python/ROOTTools.py

```diff
@@ -1581,8 +1684,27 @@ def plot_dir(x='', make=False, temp=False):
     hostname = os.environ['HOSTNAME']
     username = os.environ['USER']
     d = None
+    #if 'fnal.gov' in hostname and username == 'tucker':
+    #    if temp:
+    #        d = '/publicweb/t/tucker/asdf/tempplots'
+    #    else:
+    #        d = '/publicweb/t/tucker/asdf/plots'
+    #elif 'fnal.gov' in hostname and username == 'jchu':
+    #    d = '/publicweb/j/jchu/plots'
+    #elif 'fnal.gov' in hostname and username == 'dquach':
+    #    d = '/publicweb/d/dquach/plots'
+    #elif 'fnal.gov' in hostname and username == 'shogan':
+    #    d = '/publicweb/s/shogan/images'
+    #elif 'fnal.gov' in hostname and username == 'joeyr':
+    #    d = '/publicweb/j/joeyr/plots'
+    #elif 'fnal.gov' in hostname and username == 'ali':
+    #    d = '/publicweb/a/ali/'
+    #elif 'fnal.gov' in hostname and username == 'pkotamni':
+    #    d = '~/nobackup/crabdirs/'
+    #elif 'fnal.gov' in hostname and username == 'alecduqu':
+    #    d = '~/crab_dirs/'        Alec removed this and added the two lines below
     if 'fnal.gov' in hostname :
-        d = os.environ['HOME']+'/nobackup/DVplots/'
+        d = '~/crab_dirs/'
     if d:
```

Comment:
@duquale 

I think the 
        d = os.environ['HOME']+'/nobackup/DVplots/'
is more portable. The crab directories is mainly for your job outputs, while this is for plots!
(+ I think many people named it crabdirs instead of crab_dirs)